django/applications/catmaid/control/volume.py

```diff
@@ -14,16 +18,12 @@
 from catmaid.models import UserRole, Project, Volume
 from catmaid.serializers import VolumeSerializer
 
-from django.db import connection
-from django.http import JsonResponse, HttpResponse, HttpResponseBadRequest
-from django.shortcuts import get_object_or_404
-
 from rest_framework import renderers
 from rest_framework.decorators import api_view, renderer_classes
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
-
+#TODO: bbox_re is used by two functions - find a better way to package it than a global
```

Comment:
Something like this might be better:
```
from types import SimpleNamespace

_config = SimpleNamespace()
setattr(_config, 'num', '[-+]?[0-9]*.?[0-9]+')
setattr(_config, 'bbox_re', r'BOX3D\(({0})\s+({0})\s+({0}),\s*({0})\s+({0})\s+({0})\)'.format(_config.num))
```
And then reference `bbox_re` as `_config.bbox_re`. Or do you have an alternative suggestions?

Reply:
If we're defining a regex once per server uptime, and using it in a few places, it might be marginally more ergonomic to `re.compile` it and use the compiled version in the functions.

I personally don't have an issue with leaving it as a free variable; it's not mutable and is already namespaced to the module so I don't think there's much of an disadvantage to leaving it as a "global". I suppose prepending an underscore fits more with the scheme we currently have for non-exported objects.